setup.py

```diff
@@ -1,7 +1,6 @@
 from __future__ import print_function
-import distutils
-from distutils.core import setup, Extension, Command
-from distutils.command.build_ext import build_ext
+from setuptools import setup, Extension, Command
+from setuptools.command.build_ext import build_ext
```

Comment:
I know you hate setuptools. Sorry. This is not needed for the code to work, but it got rid of a warning about an ignored option (`install_requires`). This option makes the code play nicer with `pip` and so it might be time to switch. :(

Reply:
this is like when a bill gets written that everyone wants, but then one of the parties writes in something extra that people don't want because they think the bill will get passed no matter what :)